dask_jobqueue/core.py

```diff
@@ -280,13 +282,16 @@ def job_file(self):
                 f.write(self.job_script())
             yield fn
 
+    def submit_job(self, script_filename):
+        return self._call(shlex.split(self.submit_command) + [script_filename])
```

Comment:
This should probably be a private method so that users don't get the wrong idea that they should use it to submit jobs.